[2022-10-18 05:54:54,063] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: user_processing.process_user scheduled__2022-10-17T00:00:00+00:00 [queued]>
[2022-10-18 05:54:54,077] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: user_processing.process_user scheduled__2022-10-17T00:00:00+00:00 [queued]>
[2022-10-18 05:54:54,081] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-18 05:54:54,083] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-10-18 05:54:54,085] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-18 05:54:54,112] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): process_user> on 2022-10-17 00:00:00+00:00
[2022-10-18 05:54:54,125] {standard_task_runner.py:52} INFO - Started process 1291 to run task
[2022-10-18 05:54:54,141] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'process_user', 'scheduled__2022-10-17T00:00:00+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpk8xdldrk', '--error-file', '/tmp/tmpinuugru1']
[2022-10-18 05:54:54,145] {standard_task_runner.py:80} INFO - Job 60: Subtask process_user
[2022-10-18 05:54:54,222] {task_command.py:369} INFO - Running <TaskInstance: user_processing.process_user scheduled__2022-10-17T00:00:00+00:00 [running]> on host 5282528dd996
[2022-10-18 05:54:54,308] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=process_user
AIRFLOW_CTX_EXECUTION_DATE=2022-10-17T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-17T00:00:00+00:00
[2022-10-18 05:54:54,372] {python.py:173} INFO - Done. Returned value was: None
[2022-10-18 05:54:54,389] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=process_user, execution_date=20221017T000000, start_date=20221018T055454, end_date=20221018T055454
[2022-10-18 05:54:54,444] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-18 05:54:54,500] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
